lib/slack_main.rb

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env ruby
+
+require "dotenv"
+require "pry"
+require "httparty"
+require "terminal-table"
+
+require_relative "recipient"
+require_relative "channel"
+require_relative "user"
+require_relative "workspace"
+
+Dotenv.load
+
+CHANNEL_URL = "https://slack.com/api/channels.list"
+USER_URL = "https://slack.com/api/users.list"
```

Comment:
You have these constants defined a few places. In general, each constant should be defined once. This makes your code easier to change, and avoids the problem of what to do if they don't have the same value.